pages/system/inputs.js

```diff
@@ -102,19 +122,31 @@ _handleChange = (e) => {
         <System.Input
           label="Max length is 14"
           max={14}
-          name="sixteen"
-          value={this.state.sixteen}
+          name="exampleThree"
+          value={this.state.exampleThree}
           onChange={this._handleChange}
         />
         <br />
         <System.CodeBlock>
-          {`<System.Input
-  label="Max length is 14"
-  max={14}
-  name="sixteen"
-  value={this.state.sixteen}
-  onChange={this._handleChange}
-/>`}
+          {`class ExampleMax extends React.Component {
+   state = { exampleThree: null }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`